crates/era-cheatcodes/src/cheatcodes.rs

```diff
@@ -299,6 +311,15 @@ impl<S: DatabaseExt + Send, H: HistoryMode> DynTracer<EraDb<S>, SimpleMemory<H>>
                 prev_exception_handler_pc.replace(current.exception_handler_location);
             }
         }
+        if let Some(ActionOnReturn::Revert { prev_exception_handler_pc, .. }) =
```

Comment:
Technically this is unnecessary: you only need the exception handler and you can retrieve it anywhere and regardless of the opcode in use. 
It's done above for `is_to_label`, mostly

Reply:
and below you aren't checking for the `Ret` opcode, so looks like we can revert anytime

Reply:
Changed it to check for the `Ret` opcode now.